### ISSUE TITLE
File color changed and diff count reported in explorer for file that has no pending changes

### ISSUE THREAD

**Issue by bdharrington7** [streetsidesoftware/vscode-spell-checker#183]
**VScode version:** 1.20.1
**spell-checker version:** 1.6.3

I'm not entirely sure why *this* plugin is causing this issue, but when I disable this plugin, the problem doesn't manifest, and as soon as I re-enable it, the problem comes back.

When I open this file (which had pending changes, but I've committed them a while ago), the explorer highlights it green and shows a diff count, when there are no pending changes. I've restarted, enabled and disabled the various plugins I have installed, and this is definitely the one causing it. 

![screen shot 2018-02-16 at 10 13 56 am](https://user-images.githubusercontent.com/4553795/36322450-20dae322-1302-11e8-9abd-f037516e5f84.png)

**Comment by Jason3S** [comment]
Thank you for pointing this out.

I too just recently noticed the new behavior. They have made a change to the file explorer to show green if there are ANY issues in a file (not just spelling). It is no longer just Git file changes.

If you do not like the behavior, I think this setting controls it:

```js
  // Show Errors & Warnings on files and folder.
  "problems.decorations.enabled": true,
```

**Comment by Jason3S** [comment]
Duplicate of #181 

Setting `"problems.decorations.enabled": false,` works for me.

I'm leaving this issue open for a while incase someone else want to fix the same issue.
